src/hgraph/_types/_time_series_types.py

```diff
@@ -97,10 +97,9 @@ def modified(self) -> bool:
     @abstractmethod
     def valid(self) -> bool:
         """
-        Is there a valid value associated to this time-series input, or loosely, "has this property
-        ever ticked?". Note that it is possible for the time-series to become invalid after it has been made valid.
-        The invalidation occurs mostly when working with REF values.
-        :return: True if there is a valid value associated with this time-series.
+        Is there a valid value associated to this time-series input, or more generally has this property
+        ever ticked.
+        :return: True if there is a valid value associated to this time-series
         """
 
     @property
```

Comment:
Seems like a reversion on the docs, or was this intentional?

Reply:
No not intentional.  We do see some odd things with git using multiple remotes. My commit didn't include that change in the local repo!  I'll revert locally and include with the next push.  Thanks